Header header

BatteryBackupState backup_battery_failure_state
BatteryFailsafeState failsafe_battery_failure_state
PowerDistributionRelayState distribution_relay_state
PowerDistributionLineFailure distribution_lines_failure_state

float32 left_main_engine_generator_output_current
float32 left_main_engine_generator_output_voltage

float32 right_main_engine_generator_output_current
float32 right_main_engine_generator_output_voltage

float32 left_ballonet_control_current
float32 right_ballonet_control_current

float32 power_section_28v_power_equipment_current
float32 tail_section_28v_power_equipment_current

float32 bus_backup_28v_voltage
float32 failsafe_28v_voltage

float32 backup_battery_voltage
float32 failsafe_battery_voltage

float32 backup_battery_charge_current
float32 backup_battery_discharge_current
float32 backup_battery_monoblock_temperature


float32 failsafe_battery_charge_current
float32 failsafe_battery_discharge_current
float32 failsafe_battery_monoblock_temperature


float32 left_charge_device_output_current
float32 left_charge_device_state # reserved
float32 right_charge_device_output_current
float32 right_charge_device_state # reserved

float32 load_equipment_28v_current
